Bullwinkle.class.nut

```diff
@@ -242,11 +246,14 @@ class Bullwinkle {
         if (handler != null) {
 
             // Invoke the handler and delete the package when done
-            imp.wakeup(0, function() {
+            local timer = imp.wakeup(0, function() {
```

Comment:
Seems like we can move this pattern local timer = imp.wakeup(0, function() {}); if (timer == null) to a separate function and feed it the callback function as parameter. Would save some space and will let up easily maintain this code in the future.